<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingConfigWidget</class>
 <widget class="QWidget" name="TrainingConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Information Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Dimensions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>651</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold italic 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Recording Configuration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txDatasetName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>don't use space char here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txInfoText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>this may contain space char</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="gesturesTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>591</width>
     <height>251</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>122</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gesture Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Required Sample</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Command</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>forehand</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>do forehand shot</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>backhand</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>do backhand shot</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>service</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>do service shot</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btAdd">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btRemove">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>690</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Start Recording Now</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="num_dimension">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
